### DISCUSSION TITLE
Feature request: Improved volume indicator

### DISCUSSION THREAD

**Discussion by pgzh** [jcorporation/myMPD#897] [Ideas]
Hi,

would it be possible to (optionally) show the volume of the current output as text (e.g. "30%") instead of the icon?
The icon changes depending on volume, but one can only estimate roughly what the volume is set to with the icon.
It would be nice to be able to configure the volume indicator as icon only/percentage/icon+percentage depending on preference.

I'd really appreciate to see this implemented.

Thanks a lot for actively developing and improving this great mpd client!

best regards,
Peter

**Comment by jcorporation** [comment]
I am unsure if this is really an improvement. It clobbers the interface and I do not find the volume level so important.

**Comment by pgzh** [comment]
It's probably a personal preference similar to the battery level of a smartphone. On most phones one can choose to display the battery level as icon, text or icon with text.
I agree some people will prefer the icon to keep the interface clean, but others like me will prefer to be able to quickly see the numbers.

And basically it's the same with the current playback position - most players show the time played or remaining time and track duration. In some way it also just clobbers the ui and isn't too important mostly, but I also prefer to see the numbers as well as having a bar that shows the current position in the track.

So a configurable choice in the settings defaulting to icon only for the volume indicator would be great in my opinion.

Thank you for your consideration.

**Comment by jcorporation** [comment]
Feature is implemented in the v10.2.0 branch. The volume level text can be enabled in the settings modal.

**Reply by pgzh** [reply]
Thank you very much for reconsidering my request and also for implementing it so quickly - I'll try it asap and am looking forward to the 10.2 release!
